staging/src/k8s.io/apiserver/pkg/util/flowcontrol/apf_filter.go

```diff
@@ -172,7 +172,7 @@ func (cfgCtlr *configController) Handle(ctx context.Context, requestDigest Reque
 	defer func() {
 		klog.V(7).Infof("Handle(%#+v) => fsName=%q, distMethod=%#+v, plName=%q, isExempt=%v, queued=%v, Finish() => panicking=%v idle=%v",
 			requestDigest, fs.Name, fs.Spec.DistinguisherMethod, pl.Name, isExempt, queued, panicking, idle)
-		if idle {
+		if idle && !panicking {
```

Comment:
We explicitly considered and rejected this behavior, see https://github.com/kubernetes/kubernetes/pull/97206#issuecomment-743688255 .

#111852 reports multiple mysteries, the problem likely lies in one or more of those.